### PR TITLE
chore: better fitting of the plt title and summary.

### PR DESCRIPTION
**Pull request by sayakpaul**
@fchollet this PR does the following:

* Changes the summary of the example. The earlier one was broken.
* Adds a line of code to fit the image captions as plot titles better. 

Since the changes are minor, I decided to add all files in the PR directly. Let me know in case of anything.

### CODE REVIEW

File: examples/generative/md/finetune_stable_diffusion.md [hunk 1]
```diff
@@ -517,12 +532,12 @@ diffusion_ft_trainer.fit(training_dataset, epochs=epochs, callbacks=[ckpt_callba
 
 <div class="k-default-codeblock">
 ```
-WARNING:tensorflow:From /opt/conda/lib/python3.7/site-packages/tensorflow/python/util/deprecation.py:629: calling map_fn_v2 (from tensorflow.python.ops.map_fn) with dtype is deprecated and will be removed in a future version.
+WARNING:tensorflow:From /opt/conda/envs/py38/lib/python3.8/site-packages/tensorflow/python/util/deprecation.py:629: calling map_fn_v2 (from tensorflow.python.ops.map_fn) with dtype is deprecated and will be removed in a future version.
```
Comment (fchollet) [inline examples/generative/md/finetune_stable_diffusion.md:535]: Please remove unnecessary warning

File: examples/generative/md/finetune_stable_diffusion.md [hunk 2]
```diff
@@ -494,6 +506,9 @@ diffusion_ft_trainer.compile(optimizer=optimizer, loss="mse")
 ```
 INFO:tensorflow:Mixed precision compatibility check (mixed_float16): OK
 Your GPU will likely run quickly with dtype policy mixed_float16 as it has compute capability of at least 7.0. Your GPU: NVIDIA A100-SXM4-40GB, compute capability 8.0
+WARNING:tensorflow:From /opt/conda/envs/py38/lib/python3.8/site-packages/tensorflow/python/autograph/pyct/static_analysis/liveness.py:83: Analyzer.lamba_check (from tensorflow.python.autograph.pyct.static_analysis.liveness) is deprecated and will be removed after 2023-09-23.
+Instructions for updating:
+Lambda fuctions will be no more assumed to be used in the statement where they are used, or at least in the same block. https://github.com/tensorflow/tensorflow/issues/56089
```
Comment (fchollet) [inline examples/generative/md/finetune_stable_diffusion.md:511]: Please remove unnecessary warnings

File: examples/generative/md/finetune_stable_diffusion.md [hunk 3]
```diff
@@ -610,21 +621,21 @@ for prompt in outputs:
...
+    
 
 
 
-
-
+    
 ![png](/img/examples/generative/finetune_stable_diffusion/finetune_stable_diffusion_28_2.png)
-
+
```
Comment (fchollet) [inline examples/generative/md/finetune_stable_diffusion.md:638]: Please remove whitespace

File: examples/generative/md/finetune_stable_diffusion.md [hunk 4]
```diff
@@ -610,21 +621,21 @@ for prompt in outputs:
 ```
 
 
-
+
```
Comment (fchollet) [inline examples/generative/md/finetune_stable_diffusion.md:624]: Please remove whitespace

File: examples/generative/md/finetune_stable_diffusion.md [hunk 5]
```diff
@@ -317,15 +328,16 @@ for i in range(3):
...
 
     plt.axis("off")
 ```
 
 
-
+    
 ![png](/img/examples/generative/finetune_stable_diffusion/finetune_stable_diffusion_15_0.png)
-
+
```
Comment (fchollet) [inline examples/generative/md/finetune_stable_diffusion.md:340]: Please remove whitespace

File: examples/generative/md/finetune_stable_diffusion.md [hunk 6]
```diff
@@ -298,6 +305,10 @@ for k in sample_batch:
 
 <div class="k-default-codeblock">
 ```
+WARNING:tensorflow:Using a while_loop for converting RngReadAndSkip cause there is no registered converter for this op.
+WARNING:tensorflow:Using a while_loop for converting Bitcast cause there is no registered converter for this op.
+WARNING:tensorflow:Using a while_loop for converting Bitcast cause there is no registered converter for this op.
+WARNING:tensorflow:Using a while_loop for converting StatelessRandomUniformV2 cause there is no registered converter for this op.
```
Comment (fchollet) [inline examples/generative/md/finetune_stable_diffusion.md:311]: Please remove unnecessary warnings

File: examples/generative/md/finetune_stable_diffusion.md [hunk 7]
```diff
@@ -95,6 +96,12 @@ from keras_cv.models.stable_diffusion.text_encoder import TextEncoder
 from tensorflow import keras
 ```
 
+<div class="k-default-codeblock">
+```
+You do not have pycocotools installed, so KerasCV pycoco metrics are not available. Please run `pip install pycocotools`.
+
+```
+</div>
```
Comment (fchollet) [inline examples/generative/md/finetune_stable_diffusion.md:104]: Please remove unnecessary div

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Comment by sayakpaul** [comment]
@fchollet addressed your comments. See if the changes are good to go.

**Review by fchollet — APPROVED** [review]
LGTM, thank you!
